R/api.R

```diff
@@ -50,17 +50,31 @@ crunchAPI <- function(
 crGET <- function(...) crunchAPI("GET", ...)
 #' @rdname http-methods
 #' @export
-crPUT <- function(...) crunchAPI("PUT", ...)
+crPUT <- function(url, config = list(), ..., body) {
```

Comment:
arg order is weird here, but trying to match `crunchAPI()` so that I don't make any breaking changes

Reply:
Does body need to be named as opposed to a part of ...?

Reply:
This is the easiest way I know how to avoid evaluating the arguments inside `...` at the wrong time, I want to just pass them onto the next function as is, but eg `list(...)` evaluates all of them. (Technically I could use `...names()` & `...n()` to only evaluate body, but it looks uglier)